lib/light.js

```diff
@@ -1,32 +1,29 @@
-var Board = require("./board");
-var EVS = require("./evshield");
-var within = require("./mixins/within");
-var Fn = require("./fn");
-var Emitter = require("events").EventEmitter;
-var util = require("util");
-var priv = new Map();
-// var int16 = Fn.int16;
-var uint16 = Fn.uint16;
-var toFixed = Fn.toFixed;
-
-var Controllers = {
+const Board = require("./board");
+const EVS = require("./evshield");
+const within = require("./mixins/within");
+const { uint16, toFixed, scale } = require("./fn");
+const { EventEmitter: Emitter } = require("events");
```

Comment:
This isn't necessary, `require("events") === require("events").EventEmitter`, so you can write `const Emitter = require("events");`